scripts/utils/multisigexecutor.ts

```diff
@@ -1,22 +1,21 @@
-const hre = require("hardhat");
-const ethers = hre.ethers;
-const inquirer = require("inquirer");
+import { Contract } from "@ethersproject/contracts";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { ethers } from "hardhat";
+import inquirer from "inquirer";
+import multisigAbi from "./multisig.json";
 
-const multisigAbi = require('./multisig.json');
+export class MultisigExecutor {
+  private _deployer!: SignerWithAddress;
```

Comment:
`!` shouldn't be needed. If you don't specify it as `?` TS will require you to assign it in the constructor

Reply:
The thing is that I didn't want to change the constructor that's use in 10+ other files in this first smaller PR (the common usage pattern is [like this](https://github.com/argentlabs/argent-trustlists/blob/develop/scripts/deploy-aave.js#L23))